compiler/ballerina-lang/spotbugs-exclude.xml

```diff
@@ -75,6 +75,15 @@
     </Match>
     <Match>
         <Class name = "org.ballerinalang.toml.model.Library"/>
+        <Bug pattern="UWF_UNWRITTEN_FIELD"/>
+    </Match>
+    <Match>
+        <Class name = "org.ballerinalang.toml.model.Repository"/>
+        <Bug pattern="UWF_UNWRITTEN_FIELD"/>
+    </Match>
+    <Match>
+        <Class name = "org.ballerinalang.toml.model.Platform"/>
+        <Bug pattern="UWF_UNWRITTEN_PUBLIC_OR_PROTECTED_FIELD"/>
```

Comment:
Why do we need these exclusions?

Reply:
Getter methods are only used for some properties. Hence spot bugs reported this bug.